hack/assets/customizations.go

```diff
@@ -178,33 +187,35 @@ func findWebhookServiceSecretName(objs []unstructured.Unstructured) map[string]s
 	return serviceSecretNames
 }
 
-func customizeDeployments(obj *unstructured.Unstructured, isOperator bool) {
+func customizeDeployments(obj *unstructured.Unstructured) {
 	deployment := &appsv1.Deployment{}
 	if err := scheme.Convert(obj, deployment, nil); err != nil {
 		panic(err)
 	}
 	deployment.Spec.Template.Spec.PriorityClassName = "system-cluster-critical"
 
 	for i := range deployment.Spec.Template.Spec.Containers {
+		container := &deployment.Spec.Template.Spec.Containers[i]
 		// Add resource requests
-		deployment.Spec.Template.Spec.Containers[i].Resources.Requests = corev1.ResourceList{
+		container.Resources.Requests = corev1.ResourceList{
 			corev1.ResourceCPU:    resource.MustParse("10m"),
 			corev1.ResourceMemory: resource.MustParse("50Mi"),
 		}
 		// Remove any existing resource limits. See: https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#resources-and-limits
-		deployment.Spec.Template.Spec.Containers[i].Resources.Limits = corev1.ResourceList{}
-		// Remove all image references, they will be substituted operator later
-		deployment.Spec.Template.Spec.Containers[i].Image = "example.com/image:tag"
-	}
-
-	if isOperator {
-		// Modify manager container command to match openshift Dockerfile
-		for i := range deployment.Spec.Template.Spec.Containers {
-			container := &deployment.Spec.Template.Spec.Containers[i]
-			if container.Name == "manager" {
-				container.Command = []string{"./bin/cluster-api-operator"}
+		container.Resources.Limits = corev1.ResourceList{}
+		// Remove all image references if they are external, they will be substituted operator later
+		if !strings.HasPrefix(container.Image, "registry.ci.openshift.org") {
+			container.Image = "example.com/image:tag"
```

Comment:
Can we insert something that doesn't look like an image? To make it more obvious this needs substituting?